shared/actions/chat/send-messages.js

```diff
@@ -42,9 +42,12 @@ function* deleteMessage(action: ChatGen.DeleteMessagePayload): SagaGenerator<any
     }
     const tlfName: string = inboxConvo.name
 
+    const parsedMessageID = lastMessageID ? Constants.parseMessageID(lastMessageID) : null
+    const clientPrev = parsedMessageID && parsedMessageID.type === 'rpcMessageID' ? parsedMessageID.msgID : 0
```

Comment:
the only valid clientPrevs are from real messages and not the other things in the stream